govcd/org.go

```diff
@@ -280,36 +290,49 @@ func (org *Org) GetVDCByHref(vdcHref string) (*Vdc, error) {
 // GetVDCByName finds a VDC by Name
 // On success, returns a pointer to the VDC structure and a nil error
 // On failure, returns a nil pointer and an error
+//
+// refresh has no effect and is kept to preserve signature
 func (org *Org) GetVDCByName(vdcName string, refresh bool) (*Vdc, error) {
-	if refresh {
-		err := org.Refresh()
-		if err != nil {
-			return nil, err
-		}
+	orgVdcList, err := org.QueryOrgVdcList()
```

Comment:
it would be faster/easier to query using VDC name in filter

Reply:
It makes sense and I changed it.